packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -182,4 +289,30 @@ contract SecurityStore is DataStore
         wallets[wallet].inheritor = who;
         wallets[wallet].lastActive = uint128(now);
     }
-}
\ No newline at end of file
+
+    function cleanRemovedGuardians(address wallet)
```

Comment:
I think we can keep pendingAdditions and pendingRemovals, addGuardian to pendingAdditions first(except the first one), removeGuardian to pendingRemovals first.
And then refine this function as CleanUpGuardians which called before each public function to move pendingRemovals to guardians array and delete pendingRemovals from guardians array, and also delete themselves.
By this, we dont need loop guardians each time to judge the effectivetime to calc the numGuaridan and only calc the guardians length plus pendingAdditions and pendingRemovals.